Directory.Packages.props

```diff
@@ -1,42 +1,43 @@
-		<PackageVersion Include="MySqlConnector"                            Version="2.0.0"                                   />
+		<PackageVersion Include="MySqlConnector"                            Version="2.1.10"                                  />
 		<PackageVersion Include="AdoNetCore.AseClient"                      Version="0.19.2"                                  />
 		<PackageVersion Include="IBM.Data.DB.Provider"                      Version="11.5.5010.4" GeneratePathProperty="true" />
-		<PackageVersion Include="System.Data.SQLite.Core"                   Version="1.0.115.5"                               />
+		<PackageVersion Include="System.Data.SQLite.Core"                   Version="1.0.116"                                 />
 		<PackageVersion Include="System.Data.SqlClient"                     Version="4.8.3"                                   />
-		<PackageVersion Include="Oracle.ManagedDataAccess"                  Version="19.13.0"                                 />
-		<PackageVersion Include="Oracle.ManagedDataAccess.Core"             Version="3.21.61"                                  />
+		<PackageVersion Include="Oracle.ManagedDataAccess.Core"             Version="3.21.61"                                 />
 		<PackageVersion Include="System.Data.Odbc"                          Version="6.0.0"                                   />
 		<PackageVersion Include="System.Data.OleDb"                         Version="6.0.0"                                   />
-		<PackageVersion Include="IBM.Data.DB2.Core"                         Version="3.1.0.500"                               />
-		<PackageVersion Include="dotMorten.Microsoft.SqlServer.Types"       Version="1.3.0"                                   />
+		<PackageVersion Include="IBM.Data.DB2.Core"                         Version="3.1.0.600"                               />
+		<PackageVersion Include="dotMorten.Microsoft.SqlServer.Types"       Version="1.5.0"                                   />
 		<PackageVersion Include="Microsoft.SqlServer.Types"                 Version="14.0.1016.290"                           />
 		<!--<PackageVersion Include="linq2db4iSeries"                       Version="2.9.0"                                   />-->
 	</ItemGroup>
 	
 	<ItemGroup Condition=" '$(TargetFramework)' == 'net461'">
-		<PackageVersion Include="Microsoft.CodeAnalysis.CSharp"   Version="2.10.0" />
-		<PackageVersion Include="Npgsql"                          Version="4.1.10" />
-		<PackageVersion Include="FirebirdSql.Data.FirebirdClient" Version="7.10.1" />
+		<PackageVersion Include="Microsoft.CodeAnalysis.CSharp"   Version="2.10.0"  />
+		<PackageVersion Include="Npgsql"                          Version="4.1.10"  />
+		<PackageVersion Include="FirebirdSql.Data.FirebirdClient" Version="7.10.1"  />
+		<PackageVersion Include="Oracle.ManagedDataAccess"        Version="19.13.0" />
```

Comment:
1. no need to move it to conditioned item group
2. this file doesn't work well with `dotnet build` if package is not in unconditioned group

Reply:
21.6.1 is locked to 4.6.2 or higher. was not aware it was netfx only, so I moved it. choice is to update to 4.6.2/4.8.1 or to use older version of oracle.manageddataaccess

Reply:
I see, let's stick to 19.14.0 then

Reply:
we release only one netfx build so it doesn't make sense to reference both versions